dataflowlauncher/parsers/cli_parsers/util_cli_parser.py

```diff
@@ -13,6 +13,9 @@ def add_arguments_group(self, parser):
         util_parsers.add_argument(
             '-b', '--bypass_prompt', action='store_true',
             help='Skip enter key press required before launching. Default: false')
+        util_parsers.add_argument(
+            '--java_runtime', type=str, default='/usr/bin/java',
+            help='Optional java runtime. Default: /usr/bin/java')
```

Comment:
Does it have to be the full path? Would it work with just `java`? It would work in both *unix and win then.
```suggestion
            '--java_runtime', type=str, default='java',
            help='Optional java runtime. Default: java')
```
If it works, then the path should be updated in `README.md`.

Reply:
For me this doesn't work, I get a `FileNotFoundError`.